composer.json

```diff
@@ -31,6 +31,7 @@
         "ext-apcu": "Allows to use APCu userland caching ~ any PHP version.",
         "ext-redis": "So you can cache into Redis servers.",
         "ext-mongo": "Allows to cache into MongoDB instances.",
+        "ext-memcache": "Allows to use Memcache distributed memory caching system",
```

Comment:
`memcache` is not a distributed memory caching system but just an older Memcached client which is the only choice for Windows machine.... `ext-memcached` is far better otherwise (much more recent, faster, etc...).